### PR TITLE
Live events json#65

### PR DESCRIPTION
**Pull request by EmpyreanMist**
Json file with information that later can be used to render information about events.

### CODE REVIEW

File: content/events.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "liveEvents": [
+    {
+      "id": "event1",
+      "title": "Rock Night",
+      "image": "../pictures/rock.png",
+      "description": "Lokala rockband spelar hela natten lång hos Kino!",
+      "times": ["2025-01-15 20:00", "2025-03-22 20:00", "2025-08-03 20:00"]
```
Comment (marcusmoller97) [inline content/events.json:8]: maybe "times" can be made as:
[
  {
     "date": "2025-01-15,
     "time": "20:00"
 }
] <br>instead?

**Review by marcusmoller97 — APPROVED** [review]
Great job! The times can maybe be changed to something like metioned in the code review. Otherwise I think the json-file contains all nesecessery information. 
Good job! 😄

**Review by kikoDevv — CHANGES_REQUESTED** [review]
The json file contains lots of useful data that we can use. Besides the merge conflict everything else looks good, Maybe you can make an update on that.

File: content/events.json (hunk 1 above)
Comment (EmpyreanMist) [reply content/events.json:8]: Thanks for your suggestion, I'll look into it :)

**Comment by EmpyreanMist** [comment]
Updated time format and looked into conflicts :)

**Review by NordicNomadicLife — APPROVED** [review]
looks good now

File: content/events.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    {
+      "id": "event1",
+      "title": "Rock Night",
+      "image": "../pictures/rock.png",
+      "description": "Lokala rockband spelar hela natten lång hos Kino!",
+      "times": [
+        { "date": "2025-01-15", "time": "20:00" },
+        { "date": "2025-03-22", "time": "20:00" },
+        { "date": "2025-08-03", "time": "20:00" }
+      ]
```
Comment (itsJoel59) [inline content/events.json:12]: Neat with the date and time in objects! 🎄

**Review by itsJoel59 — APPROVED** [review]
Looks good to me!

**Review by kikoDevv — APPROVED** [review]
Nicely done!

**Pull request merged**